Outdated functions

I noticed that a couple functions may be outdated. I'm not sure if Tock has renamed some of the elements this app interacts with or something else, but the 2 I've identified so far are:

 "Expected to find element: `[data-testid="submit-purchase-button"]`, but never found it." When changed to [data-testid="purchase-button"], it worked. 

The latest error I've encountered is "AssertionError: Timed out retrying after 10000ms: Expected to find element: `.Receipt-container--header p`, but never found it.", however I don't consider this a breaking error as a reservation is completed by this step, just not reported in the terminal.

I don't have any experience in this kind of development, so my apologies if I've used inaccurate terms above. That second error isn't something I can troubleshoot further, unfortunately, due to that lack of experience. Any help would be greatly appreciated!

--- a/cypress/e2e/book-reservation.cy.ts
+++ b/cypress/e2e/book-reservation.cy.ts
@@ -132,8 +132,8 @@ describe('book reservation', () => {
 		if (reservation.dryRun) {
 			return cy.wrap('not booked, dry run mode enabled...')
 		} else {		
-			cy.get('[data-testid="submit-purchase-button"]').click()
-			return cy.get('.Receipt-container--header p', { timeout: 10000 }).then(p => {
+			cy.get('[data-testid="purchase-button"]').click()
+			return cy.get('[data-testid="receipt-confirmation-id"]', { timeout: 10000 }).then(p => {
 				return cy.wrap(`booked! ${p.text()}`)
 			})
 		}

